tests/components/ozw/test_websocket_api.py

```diff
@@ -36,10 +37,297 @@
     ERR_NOT_FOUND,
     ERR_NOT_SUPPORTED,
 )
+from homeassistant.helpers.device_registry import (
+    DeviceEntry,
+    async_get_registry as async_get_device_registry,
+)
+from homeassistant.helpers.entity_registry import (
+    RegistryEntry,
+    async_get_registry as async_get_entity_registry,
+)
 
 from .common import MQTTMessage, setup_ozw
 
 from tests.async_mock import patch
+from tests.common import MockConfigEntry, mock_device_registry, mock_registry
+
+ZWAVE_SOURCE_NODE_DEVICE_ID = "zwave_source_node_device_id"
```

Comment:
Let's move all the migration tests into their own file. Easier to clean up in the future if it is dropped.